Makefile

```diff
@@ -242,6 +242,18 @@ test-e2e:
 test-e2e-skip-upgrade:
 	@VERSION=$(VERSION) OSMOSIS_E2E_SKIP_UPGRADE=True go test -mod=readonly -timeout=25m -v $(PACKAGES_E2E)
 
+lo-keys:
```

Comment:
Two small suggestions:

- What if we move group all these Makefile targets in a `localnet` section replacing the one below? 
- To the cost of being move verbose, I would be more explicit with the naming `lo` --> `localnet` . `make localnet-start` sounds a bit better